lib/retry.rb

```diff
@@ -0,0 +1,39 @@
+require 'active_support'
+require 'active_support/logger'
+require 'active_support/tagged_logging'
+require 'memo_wise'
```

Comment:
We require these gems manually so that we can use this in `bin/prerender`, which is not run in the Rails context. If this is ever used in the Rails context (which I think it certainly could be), these `require`s will just be no-ops.